mlrun/config/dynamic.py

```diff
@@ -63,9 +63,16 @@ def get_default_function_node_selector(self):
 
     @staticmethod
     def get_valid_function_priority_class_names():
+        valid_function_priority_class_names = []
         if not config.valid_function_priority_class_names:
-            return []
-        return list(set(config.valid_function_priority_class_names.split(",")))
+            return valid_function_priority_class_names
+
+        for priority_class_name in config.valid_function_priority_class_names.split(
```

Comment:
I would add a comment above this line:
`Manually making sure we have only unique values cause we want to keep order and using a set would lose it`